scripts/extract.py

```diff
@@ -136,4 +166,11 @@ def _filter_list(self, lst):
         for i,value in enumerate(lst):
             for filt in self.string_filter_list:
                 if filt.match(value):
-                    lst.pop(i)
\ No newline at end of file
+                    lst.pop(i)
+
+
+if __name__ == "__main__":
+    extractor = DataExtractor()
+    extractor.open_file('temp/lkab.html')
+    simple_data = extractor.extract_simple_data()
+    print(simple_data)
```

Comment:
Remove this, leftovers from debugging I assume?

Reply:
Oops! I already merged this (since @Hedlund01 approved it), I will create a new PR to fix it - but until then it shouldn't cause any problems (as it is protected by a `if __name__` guard).